### ISSUE TITLE
Screen Readers don't announce dropdown is collapsed/expanded

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor4#4007]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open CKEditor with Screen Reader other than JAWS enabled
2. Press Alt+F10 to move focus to toolbar
3. Keep Pressing Tab until you reach Styles, Font, Format or Size dropdown and press Enter once you do that. 

### Expected result

 Screen Reader should announce that dropdown is collapsed and once Enter is pressed, it should announce the it has been expanded. 

### Actual result

Except for JAWS which informs about dropdown navigation in its own way, nothing happens.
It looks like the link element of button dropdown is missing the [aria-expanded](https://www.digitala11y.com/aria-expanded-state/) attribute. 

## Other details

* Browser: Any
* OS: Any
* CKEditor version: 4.14.0
* Installed CKEditor plugins: N/A

----

By adding below code in line https://github.com/ckeditor/ckeditor4/blob/major/plugins/richcombo/plugin.js#L382 there is an improvement:
```
var linkEl = el.getElementsByTag( 'a' ).getItem( 0 );
	if( linkEl )
		linkEl.setAttribute( 'aria-expanded', state == CKEDITOR.TRISTATE_ON );	
```

ChromeVox and NVDA - announce list is collapsed/expanded
Narrator - only announced the list is collapsed
JAWS ignores the attribute but announced how to navigate through dropdown in its own way.

**Comment by jacekbogdanski** [comment]
Closed in #4148
